app/models/article.rb

```diff
@@ -364,7 +364,7 @@ def self.unique_url_error
         :created_at
       end
 
-    order(column => dir.to_sym)
+    order("#{column} #{dir}")
```

Comment:
Another concern I have is if this opens up SQL injection.  We'll need to investigate this approach.

Reply:
@jeremyf thanks for the feedback, I don't know Ruby at all I just looked up the correct answer. My goal was to highlight one possible solution.